lib/class-wp-rest-sidebars-controller.php

```diff
@@ -494,57 +324,7 @@ public function get_item_schema() {
 					'description' => __( 'Nested widgets.', 'gutenberg' ),
```

Comment:
There seems to be some missing fields. See https://developer.wordpress.org/reference/functions/register_sidebar/#parameters

- class
- before_widget
- after_widget
- before_title
- after_title